lua/harpoon/mark.lua

```diff
@@ -135,6 +157,7 @@ end
 
 M.add_file = function(file_name_or_buf_id)
     local buf_name = get_buf_name(file_name_or_buf_id)
+    log.info("add_file():", buf_name)
```

Comment:
I am having a hard time following when `info` vs `debug` vs `trace` is used.  Could we have `trace` for all function calls / arguments provided?